openrewrite/test/javascript/parser/interface.test.ts

```diff
@@ -197,12 +197,23 @@ describe('as mapping', () => {
         );
     });
 
-    test.skip('interface with optional constructor signature', () => {
+    test('interface with constructor signature with type parameters', () => {
         rewriteRun(
             //language=typescript
             typeScript(`
-                interface Constructible {
-                    new? (name: string, age: number): Person; // Interface that defines a constructor signature
```

Comment:
should not we keep that test as well?

Reply:
No, because it doesn't construct signature, it is the same as test

```
interface X {
                    find ? <R, T> (v1: R, v2: T): string;
                }
```